nnpdf_data/nnpdf_data/commondata/ATLAS_1JET_13TEV_DIF/uncertainties_wo-lumi.yaml

```diff
@@ -1346,7 +1346,7 @@ definitions:
   syst_lumi:
     description: ''
     treatment: MULT
-    type: CORR
+    type: SKIP
```

Comment:
In the spirit of making the files a bit smaller, could you also remove the lines with `syst_lumi` from this file? (that way we don't need to even have the type skip here)